build/ci/olm_ocp_verification.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+#
+# Copyright 2020 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARCH=$(uname -m)
+
+if [ "${ARCH}" !=  "ppc64le" ]; then
+  for olm_ocp_dict in deploy/olm-catalog/ibm-block-csi-operator/*/ ; do
+    echo "Validating ${olm_ocp_dict}"
```

Comment:
why `dict`? I think it's more of a `bundle`/`bundle_dir`

Reply:
done